pkg/transaction/metric/metricscollector.go

```diff
@@ -25,6 +25,12 @@ import (
 	"github.com/Axway/agent-sdk/pkg/util/log"
 )
 
+const (
+	startTimestamp = "start-timestamp"
+	endTimestamp   = "end-timestamp"
+	eventType      = "event-type"
```

Comment:
hey, consts here!

Reply:
i did this because it made sense in this file